torchvision/csrc/io/decoder/gpu/README.rst

```diff
@@ -0,0 +1,20 @@
+GPU Decoder
+===========
+
+GPU decoder depends on ffmpeg for demuxing, uses NVDECODE APIs from the nvidia-video-codec sdk and uses cuda for processing on gpu. In order to use this, please follow the following steps:
```

Comment:
On the ffmpeg side, you'll probably need to make sure the the `bsf.h` file is available before trying to compile.

It turns out that `ffmpeg` installed through the default channel (anaconda), which is 4.2.2, for some reason doesn't ship with `bsf.h`.